server/src/test/java/io/spine/server/aggregate/given/AggregateRepositoryTestEnv.java

```diff
@@ -416,9 +431,9 @@ public RuntimeException getLastException() {
     }
 
     /**
-     * An aggregate class which neither handle commands nor react on events.
+     * An aggregate class which neither handle commands nor react on events or rejections.
```

Comment:
Should probably be `... which neither handles commands nor reacts on events or rejections.`